src/ngraph/serializer.cpp

```diff
@@ -262,7 +262,8 @@ class JSONDeserializer
 };
 
 static string
-    serialize(shared_ptr<ngraph::Function> func, size_t indent, bool binary_constant_data);
+    serialize_to_json(shared_ptr<ngraph::Function> func, size_t indent, bool binary_constant_data);
```

Comment:
Do we need to keep `binary_constant_data` flag?
Maybe it is worth to remove it or add to `serialize_to_cpio` function as well?

Reply:
For json default value of  ```binary_constant_data``` is false for cpio true. In current version to both   ```serialize_to_json``` and ```serialize_to_cpio``` were proper default arguments. As a result interface is consistent.